requirements/requirements.txt

```diff
@@ -103,3 +103,4 @@ contextlib2==0.5.4
 hiredis==0.2.0
 xlwt==1.3.0
 ddtrace==0.9.2
+git+git://github.com/nickpell/django-package-monitor@np/handle-no-semver
```

Comment:
do we need this on prod?

Reply:
Good point.  I've updated the PR so the app can only be used in dev environments.